### ISSUE TITLE
[FEATURE REQUEST] `chain`?

### ISSUE THREAD

**Issue by thomasttvo** [newdash/newdash#533]
Is there no `chain` function? For example:

```ts
chain(value)
	.map(.....)
	.flatten(....)
	.value();
```

**Comment by Soontao** [comment]
yes, the `chain` is removed from `newdash`, because `chain` will `import` too much modules which conflict with the `tree shaking` optimization, and the functionality could be implemented by other way, so, I removed it.

**Comment by thomasttvo** [comment]
@Soontao I see. Do you know the commit where it was removed? Our project is `react-native` so we don't really need tree shaking too much.

That said, what do you think about a `rxjs`-like pipe syntax:

```ts
pipe(value)(
	map()
	flatten()
	orderBy()
)
```

**Comment by thomasttvo** [comment]
Ah you know what, this repo shows an example of using the pipeline operator with typescript. Sounds promising. https://github.com/Dabolus/typescript-pipeline-operator/blob/master/pipeline.ts

**Comment by Soontao** [comment]
sounds good ? let me check this later, I'm busy on the daily meetings haha
